integration-tests/pipelines/konflux-e2e-tests.yaml

```diff
@@ -44,6 +44,9 @@ spec:
     - name: enable-sl-plugin
```

Comment:
are both needed? Isn't this a duplicate? the description make it look like the same flag

Reply:
https://github.com/konflux-ci/e2e-tests/blob/7d287c1c637013e774a24364adbb1cb648285152/integration-tests/tasks/konflux-e2e-tests-task.yaml#L201  it's needed, otherwise step is skipped

the enable-sl-plugin is exported only as env var, dunno where it's used, it can be used by those go scripts, it's related to ginkgo somehow